articles/universal-login/index.md

```diff
@@ -0,0 +1,92 @@
+---
+description: Overview of Universal Login with Auth0
+topics:
+  - login
+  - universal-login
+  - password-reset
+  - guardian
+  - mfa
+  - error-pages
+  - hosted-pages
+contentType: index
+toc: true
+useCase: customize-hosted-pages
+---
+# Auth0 Universal Login
+
+Auth0's Universal Login is the most secure way to authenticate users for your applications. Universal Login centers around your Auth0 login page. The login page appearance and behavior is customizable right from the [Dashboard](${manage_url}). By default, the login page internally uses Auth0's [Lock Widget](/libraries/lock) to authenticate your users, but the code of the login page can be customized to replace the standard Lock widget with the Lock (Passwordless Mode) widget, or an entirely custom UI can be built in its place, using the [Auth0.js SDK](/libraries/auth0js) for authentication.
+
+![Login Page](/media/articles/hosted-pages/hlp-lock.png)
+
```

Comment:
In line with the comment above, I don't think the first thing we should see here is the HTML page, I'd prefer the color picker.